src/databricks/labs/ucx/cli.py

```diff
@@ -504,5 +505,43 @@ def create_catalogs_schemas(w: WorkspaceClient, prompts: Prompts):
     catalog_schema.create_catalog_schema()
 
 
+@ucx.command
+def cluster_remap(w: WorkspaceClient, prompts: Prompts):
+    """Re-mapping the cluster to UC"""
+    logger.info("Remapping the Clusters to UC")
+    installation = Installation.current(w, 'ucx')
+    cluster = ClusterAccess(installation, w, prompts)
+    cluster_list = cluster.list_cluster()
+    print("Cluster Name\tCluster Id")
+    for name, cluster_id in cluster_list.items():
+        print(f"{name}\t{cluster_id}")
```

Comment:
Can you provide a screenshot of this working as a table?

Otherwise refactor.

Reply:
<img width="1079" alt="image" src="https://github.com/databrickslabs/ucx/assets/127273819/f6baec37-8f4e-40c1-81cc-3b0b6e1482a9">
I have adjusted the spacing.